es/src/it/scala/monix/connect/es/ElasticsearchSourceSuite.scala

```diff
@@ -15,26 +16,33 @@ class ElasticsearchSourceSuite extends AnyFlatSpecLike with Fixture with Matcher
     client.execute(deleteIndex("*")).runSyncUnsafe()
   }
 
-  "ElasticsearchSource" should "fails when there is no keep alive time" in {
+  "ElasticsearchSource" should "emit searched results to downstream" in {
     // given
-    val index = "test_index"
-    val searchRequest = search(index).query(matchAllQuery())
-    val searchSettings = ElasticsearchSource.SearchSettings(searchRequest)
+    val index = genIndex.sample.get
+    val updateRequests = Gen.listOfN(1000, genUpdateRequest(index)).sample.get
+    val searchRequest = search(index).query(matchAllQuery()).keepAlive("1m")
+    val source = ElasticsearchSource.search(searchRequest)
 
     // when
-    val source = Try(ElasticsearchSource.search(searchSettings))
+    Elasticsearch.bulkRequest(updateRequests).runSyncUnsafe()
+    Elasticsearch.refresh(index).runSyncUnsafe()
 
     // then
-    source.isFailure shouldBe true
+    val r = source
+      .bufferTimedAndCounted(100.millis, 100)
+      .map(_.map(_.id))
+      .toListL
+      .runSyncUnsafe()
+      .flatten
+    r should contain theSameElementsAs updateRequests.map(_.id).distinct
```

Comment:
is `.distinct` necessary? the source only reads the same element once, right?

Reply:
The `id` of generated requests might be duplicated.